pkgdb2/templates/list_actions.html

```diff
@@ -58,7 +58,7 @@
     <tr>
         <td>
         {% if page > 1%}
-            <a href="{{ url_for('.admin_action') }}?page={{page - 1}}&package={{package}}&from_date={{from_date}}">
+            <a href="{{ url_for('.admin_actions') }}?page={{page - 1}}&package={{package}}&from_date={{from_date}}">
```

Comment:
Well, I got stuck in the pkgdb2 test suite.  This is how far I got, but I couldn't get the items I was adding to the session to actually be returned by the test app.  I tried both ``self.session`` and ``pkgdb2.SESSION``.  Giving up on this for now.

```diff
diff --git a/tests/__init__.py b/tests/__init__.py
index 7f31e89..d7022f7 100644
--- a/tests/__init__.py
+++ b/tests/__init__.py
@@ -684,7 +684,7 @@ def create_package_acl2(session):
     session.commit()
 
 
-def create_admin_actions(session):
+def create_admin_actions(session, n=1):
     """ Add an Admin Actions for the tests. """
     guake_pkg = model.Package.by_name(session, 'guake')
     el6_collec = model.Collection.by_name(session, 'el6')
@@ -699,6 +699,19 @@ def create_admin_actions(session):
 
     session.add(action)
 
+    if n > 1:
+        f17_collec = model.Collection.by_name(session, 'f17')
+
+        action = model.AdminAction(
+            package_id=guake_pkg.id,
+            collection_id=f17_collec.id,
+            user='ralph',
+            _status='Pending',
+            action='request.branch',
+        )
+
+        session.add(action)
+
     session.commit()
 
 
diff --git a/tests/test_flask_ui_admin.py b/tests/test_flask_ui_admin.py
index 57ee13e..4e449c6 100644
--- a/tests/test_flask_ui_admin.py
+++ b/tests/test_flask_ui_admin.py
@@ -36,7 +36,10 @@ sys.path.insert(0, os.path.join(os.path.dirname(
     os.path.abspath(__file__)), '..'))
 
 import pkgdb2
-from tests import (Modeltests, FakeFasUser, FakeFasUserAdmin, user_set)
+from tests import (
+    Modeltests, FakeFasUser, FakeFasUserAdmin, user_set,
+    create_collection, create_package, create_admin_actions,
+)
 
 
 class FlaskUiAdminTest(Modeltests):
@@ -54,8 +57,10 @@ class FlaskUiAdminTest(Modeltests):
         pkgdb2.ui.collections.SESSION = self.session
         pkgdb2.ui.packagers.SESSION = self.session
         pkgdb2.ui.packages.SESSION = self.session
+        pkgdb2.APP.config['ITEMS_PER_PAGE'] = 1
         self.app = pkgdb2.APP.test_client()
 
+
     @patch('pkgdb2.is_admin')
     def test_admin(self, login_func):
         """ Test the admin function. """
@@ -112,6 +117,10 @@ class FlaskUiAdminTest(Modeltests):
         """ Test the admin_actions function. """
         login_func.return_value = None
 
+        create_collection(pkgdb2.SESSION)
+        create_package(pkgdb2.SESSION)
+        create_admin_actions(pkgdb2.SESSION, n=2)
+
         user = FakeFasUserAdmin()
         with user_set(pkgdb2.APP, user):
             output = self.app.get('/admin/actions/')
@@ -121,6 +130,9 @@ class FlaskUiAdminTest(Modeltests):
                 'Restrict to package: <input type="text" name="package" />'
                 in output.data)
 
+            # TODO -- test here that only one showed up on the page but not the
+            # other.. and that pagination didn't create a 500 error
+
             output = self.app.get(
                 '/admin/actions/?page=abc&limit=def&status=ghi&package=test')
             self.assertEqual(output.status_code, 200)
```